changelog/fragments/helm-upgrade-rename.yaml

```
@@ -0,0 +1,5 @@
+entries:
+  - description: >
+      In Helm-based operators, usages of the verb `Update` were renamed to `Upgrade` to better align with Helm nomenclature.
```

The only change that affects the end-users here is the log.info. In this case, we might do not need the fragment, however, if we keep I'd recommend. 

```suggestion
      In Helm-based operators, usages of the verb `Update` in the logs were renamed to `Upgrade` to better align with Helm nomenclature.
```

```suggestion
    breaking: false
```